opentelemetry-prometheus/src/lib.rs

```diff
@@ -227,6 +284,8 @@ impl PrometheusExporter {
         controller: PullController,
         default_summary_quantiles: Vec<f64>,
         default_histogram_boundaries: Vec<f64>,
+        host: String,
+        port: String,
```

Comment:
6 parameters seem a lot in this case. Should we consider deprecate this method in favor of the builder path?

Reply:
I agree that it felt a little long...the builder pattern feels more consistent with the other exporters.  How does something like this sound?

```rust
#[deprecated(
    since = "v0.9.0",
    note = "Please use the ExporterBuilder to initialize a PrometheusExporter"
)]
```